app/views/provider/admin/dashboards/show.html.slim

```diff
@@ -20,14 +21,22 @@
         // Account Level Widgets
         = dashboard_widget :new_accounts
         = dashboard_widget :potential_upgrades if can?(:manage, :plans)
+    section.DashboardSection.DashboardSection--audience class=('DashboardSection--wide' unless can?(:manage, :plans))
+      - if current_user.accessible_services.size > 1
```

Comment:
Hmm do not use `size` as it makes a count in DB
Prefer using `current_user.accessible_services?`

```ruby

def accessible_services?
  accessible_services.exists?
end
```